eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -1,19 +1,27 @@
 parameters:
   Directory: 'not-specified'
-  IgnoreLinksFile: "$(Build.SourcesDirectory)/eng/ignore-links.txt"
-  
+  IgnoreLinksFile: '$(Build.SourcesDirectory)/eng/ignore-links.txt'
+  WorkingDirectory: '$(System.DefaultWorkingDirectory)'
+  ScriptDirectory: 'eng/common/scripts'
+  Recursive: $false
+  CheckLinkGuidance: $true
```

Comment:
Might be better to start by defaulting this to $false and get the common engineering system changes in and then pass true in each repo as you enable the feature.

Reply:
Thanks! I will turn this feature on in each repo, and then enable this in azure-sdk-tool.